cfme/tests/automate/test_method.py

```diff
@@ -264,3 +265,96 @@ def test_task_id_for_method_automation_log(request, generic_catalog_item):
     wait_for(lambda: service_request.request_state == "active", fail_func=service_request.reload,
              timeout=60, delay=3)
     result.validate_logs()
+
+
+@pytest.mark.meta(blockers=[BZ(1704439)])
+@pytest.mark.meta(server_roles="+notifier")
+def test_send_email_method(request, klass):
+    """
+    Polarion:
+        assignee: ghubale
+        initialEstimate: 1/20h
+        caseimportance: high
+        testtype: functional
+        startsin: 5.10
+        casecomponent: Automate
+
+    Bugzilla:
+        1688500
+        1702304
+    """
+    # Resetting role - 'notifier' to default value
+    request.addfinalizer(
+        lambda: klass.appliance.server.settings.update_server_roles_ui({"notifier": False})
+    )
+
+    mail_to = fauxfactory.gen_email()
+    mail_cc = fauxfactory.gen_email()
+    mail_bcc = fauxfactory.gen_email()
+
+    script = '''
+                \nto = "{mail_to}"
+                \nsubject = "Hello"
+                \nbody = "Hi"
+                \nbcc = "{mail_bcc}"
+                \ncc = "{mail_cc}"
+                \ncontent_type = "message"
+                \nfrom = "cfadmin@cfserver.com"
+             '''
+    script = script.format(mail_cc=mail_cc, mail_bcc=mail_bcc, mail_to=mail_to)
+    script += (
```

Comment:
Q; why not it's part of the script directly?

Reply:
I tried adding this part to script it self but while ```formatting``` that multi-line string it gives error and leads to change in syntax of ruby code. Hence updated as above.